dotnet-roslyn/config/Audacia.CodeAnalysis/Base/.editorconfig

```diff
@@ -2233,4 +2233,283 @@ dotnet_diagnostic.IDE0051.severity = none
+
+# Use pattern matching to avoid `is` check followed by a cast (with variable)
+dotnet_diagnostic.IDE0020.severity = warning
+
+# Use expression body for constructors
+dotnet_diagnostic.IDE0021.severity = warning
+
+# Use expression body for methods
+dotnet_diagnostic.IDE0022.severity = warning
+
+# Use expression body for conversion operators
+dotnet_diagnostic.IDE0023.severity = warning
+
+# Use expression body for operators
+dotnet_diagnostic.IDE0024.severity = warning
+
+# Use expression body for properties
+dotnet_diagnostic.IDE0025.severity = warning
+
+# Use expression body for indexers
+dotnet_diagnostic.IDE0026.severity = warning
+
+# Use expression body for accessors
+dotnet_diagnostic.IDE0027.severity = warning
+
+# Use collection initializers
+dotnet_diagnostic.IDE0028.severity = warning
+
+# Use coalesce expression (non-nullable types)
+dotnet_diagnostic.IDE0029.severity = warning
+
+# Use coalesce expression (nullable types)
+dotnet_diagnostic.IDE0030.severity = warning
+
+# Use null propagation
+dotnet_diagnostic.IDE0031.severity = warning
+
+# Use auto property
+dotnet_diagnostic.IDE0032.severity = warning
+
+# Use explicitly provided tuple name
+dotnet_diagnostic.IDE0033.severity = warning
+
+# Simplify default expression
+dotnet_diagnostic.IDE0034.severity = warning
+
+# Remove unreachable code
+dotnet_diagnostic.IDE0035.severity = warning
+
+# Order modifiers
+dotnet_diagnostic.IDE0036.severity = warning
+
+# Use inferred member name
+dotnet_diagnostic.IDE0037.severity = warning
+
+# Use pattern matching to avoid is check followed by a cast (without variable)
+dotnet_diagnostic.IDE0038.severity = warning
+
+# Use local function instead of lambda
+dotnet_diagnostic.IDE0039.severity = warning
+
+# Add accessibility modifiers
+dotnet_diagnostic.IDE0040.severity = warning
+
+# Use is null check
+dotnet_diagnostic.IDE0041.severity = warning
+
+# Deconstruct variable declaration
+dotnet_diagnostic.IDE0042.severity = warning
+
+# Add readonly modifier
+dotnet_diagnostic.IDE0044.severity = warning
+
+# Use conditional expression for assignment
+dotnet_diagnostic.IDE0045.severity = warning
+
+# Use conditional expression for return
+dotnet_diagnostic.IDE0046.severity = warning
+
+# Remove unnecessary parentheses
+dotnet_diagnostic.IDE0047.severity = warning
+
+# Add parentheses for clarity
+dotnet_diagnostic.IDE0048.severity = warning
+
+# Use language keywords instead of framework type names for type references
+dotnet_diagnostic.IDE0049.severity = warning
+
+# Convert anonymous type to tuple
+dotnet_diagnostic.IDE0050.severity = warning
+
+# Remove unused private member
+dotnet_diagnostic.IDE0051.severity = warning
+
+# Remove unread private member
+dotnet_diagnostic.IDE0052.severity = warning
+
+# Use expression body for lambdas
+dotnet_diagnostic.IDE0053.severity = warning
+
+# Use compound assignment
+dotnet_diagnostic.IDE0054.severity = warning
+
+# Fix formatting
+dotnet_diagnostic.IDE0055.severity = warning
```

Comment:
Probably a separate piece of work, but should we also be defining both the .NET and C# formatting options that this rule would warn about?

https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0055

Reply:
I'll create a PBI in the internal development backlog for us to review.